quickstarts/manage-data/airflow-airbyte-dbt.md

```diff
@@ -0,0 +1,243 @@
+---
+sidebar_position: 5.5
+title: Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+author: Vidhan Bhonsle
+email: vidhan.bhonsle@teradata.com
+page_last_update: October 4th, 2024
+description: Using Airflow to Orchestrate Airbyte and dbt for Teradata
+keywords: [data warehouses, airflow, teradata, vantage, orchestration, object storage, airbyte, enterprise analytics, elt, dbt.]
+---
+
+import ClearscapeDocsNote from '../_partials/vantage_clearscape_analytics.mdx'
+import CommunityLink from '../_partials/community_link.mdx'
+import ProfileTabs from '../_partials/tabsDBTProfiles.mdx'
+
+# Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+
+## Overview
+
+This tutorial demonstrates how to leverage Apache Airflow as orchestration tool to automate the movement of data from a source to Teradata using Airbyte. Once the data is in Teradata, trigger dbt to perform transformations, ensuring the data is clean, reliable and ready for analysis.
+
+## Prerequisites
+
+* Access to a Teradata Vantage instance.
+
+    <ClearscapeDocsNote />
+
+* Python **3.10** or later installed.
+
+* [Docker and Docker Compose (Docker Desktop)](https://docs.docker.com/get-started/get-docker/)
```

Comment:
I don't think Docker Compose is needed if you go with `abctl` as it deploys to Kubernetes?

Reply:
Docker Desktop is the prerequisite for abctl. (https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart#prerequisites)

Reply:
Right, I meant only Docker Compose, but I see we need it for Airflow.